misc.c

```diff
@@ -1457,7 +1457,12 @@ ValidateManagementDaemon(connection_t *c)
     wchar_t win32_path[MAX_PATH];
     swprintf(win32_path, _countof(win32_path), L"%ls%ls", L"\\\\?\\GLOBALROOT", nt_path);
 
+#ifdef ENABLE_OVPN3
+    TCHAR* exe_path = o.ovpn_engine == OPENVPN_ENGINE_OVPN3 ? o.omi_exe_path : o.exe_path;
```

Comment:
Please use WCHAR or wchar_t instead of TCHAR for all new code. Anyway this needs a v2 to make clang-format happy.